repos/system_upgrade/el7toel8/models/ipuconfig.py

```diff
@@ -0,0 +1,11 @@
+from leapp.models import Model, fields
+from leapp.topics import SystemInfoTopic
+
+
+class IPUConfig(Model):
+    """
+    IPU workflow configuration model
+    """
+    topic = SystemInfoTopic
+
+    leapp_env_vars = fields.List(fields.JSON(), default=[])
```

Comment:
Why a list of JSON? This doesn't seem right. There are two things I can imagine:

`fields.List(fields.Model(EnvVar))`
or just `fields.JSON()`

Where EnvVar would be:

```python
class EnvVar(Model):
     topic = ...
     name = fields.String()
     value = fields.String()
```

Reply:
and to be straight, I prefer the List(EnvVar) approach